util/ubuntuinstall.bash

```diff
@@ -1,32 +1,37 @@
 #!/bin/bash
 
-echo "This script installs deps for running depserver on a Ubuntu server."
-
-# base/certbot ppa
-echo "Getting updates"
-sudo apt-get install software-properties-common
-sudo add-apt-repository universe
-sudo add-apt-repository ppa:certbot/certbot
-sudo apt-get update
+echo "This script will now install deps on your Ubuntu server..."
 
 # packages
-echo "Installing updates"
-sudo apt install mosh python3-pip certbot
+echo "Installing python, certbot and other DNS related libraries..."
+sudo apt -yq install mosh python3-pip certbot
 sudo pip3 install dnslib cherrypy
 
 # kill resolved
-echo "Removing resolved"
-echo "127.0.0.1 $(hostname)" >> /etc/hosts 
-sudo systemctl disable systemd-resolved
-sudo systemctl stop systemd-resolved
-rm -f /etc/resolv.conf
-echo "nameserver 127.0.0.1" > /etc/resolv.conf
+echo "Stopping resolved from using port 53..."
+
+# thanks https://www.linuxuprising.com/2020/07/ubuntu-how-to-free-up-port-53-used-by.html
+sudo cat > /etc/systemd/resolved.conf << EOF
+[Resolve]
+DNS=1.1.1.1
+#FallbackDNS=
+#Domains=
+#LLMNR=no
+#MulticastDNS=no
+#DNSSEC=no
+#DNSOverTLS=no
+#Cache=no
+DNSStubListener=no
+#ReadEtcHosts=yes
+EOF
+sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
 
+echo "Securing system with ufw..."
 sudo ufw allow ssh
 sudo ufw allow 53
 sudo ufw allow 80
 sudo ufw allow 443
 sudo ufw allow 60000:61000/udp
 sudo ufw enable
 
-echo "Ready. Now run python3 dnsserver.py"
\ No newline at end of file
+echo "Please reboot your system after which you are ready to run python3 dnsserver.py"
```

Comment:
any particular reason to reboot? Just restarting the service should be enough, no?

Reply:
I agree! Full reboots are kinda a pain.  However, the [guide I cited in the comment](https://github.com/Corollarium/localtls/pull/6/files#diff-734269221ffeb8f00d8beadbbc966ba61930c64054cae60857434a80aff4ed98R13) suggest the reboot and in my testing I'd just followed that guide and it worked. I didn't want to document and untested path